plugin/trino-example-plugin/src/main/java/com/example/ExampleConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import io.airlift.configuration.Config;
+
+public class ExampleConfig
+{
+    private String someSetting = "default-value";
+
+    public String getSomeSetting()
+    {
+        return someSetting;
+    }
+
+    @Config("some_setting")
```

Comment:
We normally use dashes in config names. Let's change the example to make the config actually meaningful somehow and actually used, rather than just being garbage.

Reply:
Add `@ConfigDescription` as that is best practice

Reply:
All done!